CONTRIBUTING.md

```diff
@@ -1,37 +1,261 @@
+[alternative standard libraries][other-standard-libraries]).
+
+#### Maintaining existing packages ####
+
+Some Coq packages are not maintained by their initial authors anymore
+(for instance because they changed jobs, or lost interest) even if
+they were useful, or interesting.  The coq-community organization is a
+place for volunteers to take over the maintenance of such packages.
+
+If you want to contribute by becoming a maintainer, there is [a list
+of packages waiting for a
+maintainer][coq-community-maintainer-wanted].  You can also propose a
+package that is not listed.  Find out more about coq-community in [the
+manifesto's README][coq-community-manifesto].
+
+### Contributing to the wiki ###
+
+Coq's [wiki][] is an informal source of additional documentation which
+anyone with a GitHub account can edit directly.  In particular, it
+contains the Coq [FAQ][] which has not seen so many updates in the
+recent years.  You should feel free to fix it, expand it, and even
+refactor it (if you are not sure if some changes would be welcome, you
+can open an issue to discuss them before performing them).
+
+People who watch the Coq repository will see recent wiki edits in
+their GitHub feed.  It is recommended to review them *a posteriori* to
+check no mistake was introduced.  The wiki is also a standard git
+repository, so people can follow the changes using any standard git
+tool.
+
+Coq's wiki is formatted using GitHub's flavored Markdown, with some
+wiki-specific extensions.  See:
+
+- [GitHub's Markdown guide][GitHub-markdown]
+- [GitHub's wiki extensions][GitHub-wiki-extensions]
+
+### Contributing to the website or the package archive ###
+
+The website and the package archive have their own repositories:
+
+- <https://github.com/coq/www>
+- <https://github.com/coq/opam-coq-archive>
+
+You can contribute to them by using issues and pull requests on these
+repositories.  These repositories should get their own contributing
+guides, but they don't have any at the time of writing this.
+
+### Other ways of creating content ###
+
+There are many other ways of creating content and making the Coq
+community strive, including many which we might not have thought
+about.  Feel free to add more references / ideas to this sub-section.
+
+You can tweet about Coq, you can give talks about Coq both in
+academic, and in non-academic venues (such as developer conferences).
+
+[Codewars][] is a platform where people can try to solve some
+programming challenges that were proposed by other community members.
+Coq is supported and the community is eager to get more challenges.
+
+## Issues ##
+
+### Reporting a bug, requesting an enhancement ###
+
+Bug reports are enormously useful to identify issues with Coq; we
+can't fix what we don't know about.  To report a bug, please open an
+issue in the [Coq issue tracker][Coq-issue-tracker] (you'll need a
+GitHub account).  You can file a bug for any of the following:
```

Comment:
IMVHO we should mention here [but not require] to try to reproduce the bug with `master`, I'd dare to say that a good number of reported bugs are already fixed. More importantly, the minimum required version should be the stable one, it is not uncommon to see people reporting bugs with 8.8.

I think it is safe to state that the only supported Coq version is the latest stable one.